Lib/idlelib/config.py

```diff
@@ -871,7 +871,8 @@ def clear(self):
 
 
 # TODO Revise test output, write expanded unittest
-def _dump():  # htest # (not really, but ignore in coverage)
+def _dump():  # pragma: no cover
+    # htest # (not really, but ignore in coverage)
```

Comment:
Please don't undo my improvements.  This adds noise and disables the ignore feature.  _dump is a human test, but not specifically an 'htest' as defined by idle_test.htest.